src/NUnitFramework/framework/Internal/Tests/Test.cs

```diff
@@ -335,7 +335,7 @@ public void ApplyAttributesToTest(ICustomAttributeProvider provider)
         public void ApplyAttributesToTest(Type type)
         {
             foreach (var t in GetNestedTypes(type).Reverse()) 
-                ApplyAttributesToTest((ICustomAttributeProvider) t.GetTypeInfo());
+                ApplyAttributesToTest((ICustomAttributeProvider) t);
```

Comment:
This cast is probably showing as unnecessary in the IDE now?

Reply:
still required since it forces a different path